cmd/ocm-backplane/accessrequest/getAccessRequest.go

```diff
@@ -0,0 +1,109 @@
+package accessrequest
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	ocmcli "github.com/openshift-online/ocm-cli/pkg/ocm"
+	ocmsdk "github.com/openshift-online/ocm-sdk-go"
+	acctrspv1 "github.com/openshift-online/ocm-sdk-go/accesstransparency/v1"
+	"github.com/openshift/backplane-cli/pkg/ocm"
+	"github.com/openshift/backplane-cli/pkg/utils"
+	logger "github.com/sirupsen/logrus"
+)
+
+// newGetAccessRequestCmd returns cobra command
+func newGetAccessRequestCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:           "get",
+		Short:         "Get the active (pending or approved) access request",
+		Args:          cobra.ExactArgs(0),
+		SilenceUsage:  true,
+		SilenceErrors: true,
+		RunE:          runGetAccessRequest,
+	}
+
+	return cmd
+}
+
+func getClusterID(cmd *cobra.Command) (string, error) {
+	clusterKey, err := cmd.Flags().GetString("cluster-id")
+	if err != nil {
+		return "", err
+	}
+
+	bpCluster, err := utils.DefaultClusterUtils.GetBackplaneCluster(clusterKey)
+	if err != nil {
+		return "", err
+	}
+
+	return bpCluster.ClusterID, nil
+}
+
+func getAccessRequest(clusterID string, ocmConnection *ocmsdk.Connection) (*acctrspv1.AccessRequest, error) {
+	isEnabled, err := ocm.DefaultOCMInterface.IsClusterAccessProtectionEnabled(ocmConnection, clusterID)
+	if err != nil {
+		return nil, fmt.Errorf("unable to determine if access protection is enabled or not for cluster '%s': %v", clusterID, err)
+	}
+
+	if !isEnabled {
+		return nil, fmt.Errorf("access protection is not enabled for cluster '%s'", clusterID)
+	}
+
+	return ocm.DefaultOCMInterface.GetClusterActiveAccessRequest(ocmConnection, clusterID)
+}
+
+func printAccessRequest(clusterID string, accessRequest *acctrspv1.AccessRequest) {
+	accessRequestStatus := accessRequest.Status()
+	accessRequestStatusState := acctrspv1.AccessRequestState("<Undefined>")
+
+	if accessRequestStatus != nil && accessRequestStatus.State() != "" {
+		accessRequestStatusState = accessRequestStatus.State()
+	}
```

Comment:
IMO, You may able to move these logics to pkg/access-request module and write some unit tests. 
otherwise it may be hard to write tests with cobra dependencies

Reply:
Agreed, I may need to move the code a bit for unittest purposes.